HomeTask2/HomeTask2.js

```diff
@@ -1,88 +1,230 @@
+}
+
+function ClosePopUp() {
+    var modal = document.getElementById("myModal");
+    modal.style.display = "none";
 }
 
 function SelectCategory() {
-    var value = document.getElementById("category").value;
-    if (value == "All") {
-        var tittle1 = document.getElementById("tittle1");
-        var tittle2 = document.getElementById("tittle2");
-        var tittle3 = document.getElementById("tittle3");
-        var tittle4 = document.getElementById("tittle4");
-        var tittle5 = document.getElementById("tittle5");
-        tittle1.style.display = "block";
-        tittle2.style.display = "block";
-        tittle3.style.display = "block";
-        tittle4.style.display = "block";
-        tittle5.style.display = "block";
-    }
-    if (value == "TittleOne") {
-        var tittle1 = document.getElementById("tittle1");
-        var tittle2 = document.getElementById("tittle2");
-        var tittle3 = document.getElementById("tittle3");
-        var tittle4 = document.getElementById("tittle4");
-        var tittle5 = document.getElementById("tittle5");
-        tittle1.style.display = "block";
-        tittle2.style.display = "none";
-        tittle3.style.display = "none";
-        tittle4.style.display = "none";
-        tittle5.style.display = "none";
-    }
-    if (value == "TittleTwo") {
-        var tittle1 = document.getElementById("tittle1");
-        var tittle2 = document.getElementById("tittle2");
-        var tittle3 = document.getElementById("tittle3");
-        var tittle4 = document.getElementById("tittle4");
-        var tittle5 = document.getElementById("tittle5");
-        tittle1.style.display = "none";
-        tittle2.style.display = "block";
-        tittle3.style.display = "none";
-        tittle4.style.display = "none";
-        tittle5.style.display = "none";
-    }
-    if (value == "TittleThree") {
-        var tittle1 = document.getElementById("tittle1");
-        var tittle2 = document.getElementById("tittle2");
-        var tittle3 = document.getElementById("tittle3");
-        var tittle4 = document.getElementById("tittle4");
-        var tittle5 = document.getElementById("tittle5");
-        tittle1.style.display = "none";
-        tittle2.style.display = "none";
-        tittle3.style.display = "block";
-        tittle4.style.display = "none";
-        tittle5.style.display = "none";
-    }
-    if (value == "TittleFour") {
-        var tittle1 = document.getElementById("tittle1");
-        var tittle2 = document.getElementById("tittle2");
-        var tittle3 = document.getElementById("tittle3");
-        var tittle4 = document.getElementById("tittle4");
-        var tittle5 = document.getElementById("tittle5");
-        tittle1.style.display = "none";
-        tittle2.style.display = "none";
-        tittle3.style.display = "none";
-        tittle4.style.display = "block";
-        tittle5.style.display = "none";
+    const value = document.getElementById("mySelect").value;
```

Comment:
If you are using a drop down use value property to set the value. Then pass that value to another function where you will be toggling the classes of the section. you can use document.getElementById(#id).classList.add/remove functions to do that. Explore the same, Try to minimize the code